kld/tests/smoke/start.rs

```diff
@@ -181,8 +182,9 @@ pub async fn test_start() -> Result<()> {
     Ok(())
 }
 
+#[serial_test::serial]
+#[test_with::no_env(TEST_IN_FLAKE)]
```

Comment:
I only use this test when I am testing something manually like the GUI, so I just need KLD to run for a while and return test data. And I only run this specific test. I don't think its a good enough reason to add the new dependency.

Reply:
Dev dependencies do not take any effect on the release binary.  If you mind this, I remove it.